src/main/java/net/openhft/chronicle/queue/impl/single/StoreTailer.java

```diff
@@ -1209,7 +1209,7 @@ public boolean readAfterReplicaAcknowledged() {
     public long approximateExcerptsInCycle(int cycle) {
```

Comment:
should we deprecate both of these and introduce an `excerptsInCycle` method?

Reply:
Done